myokit/tests/test_formats_importers.py

```diff
@@ -50,6 +50,12 @@ def test_unknown(self):
 
 class SBMLTest(unittest.TestCase):
     """ Test SBML import. """
+    # Test Case I: Hodgkin Huxley
+    i = formats.importer('sbml')
+    HoHu = i.model(os.path.join(
```

Comment:
Could we call this something in camel case?

Reply:
Yep, done.